test/flint/padic-test.jl

```diff
@@ -1,3 +1,17 @@
+function test_elem(R::FlintPadicField)
+   p = prime(R)
+   prec = rand(1:R.prec_max)
+   r = fmpz(0):p-1
+   return R(sum(rand(r)*p^i for i in 0:prec))
+end
+
+@testset "padic.conformance_tests" begin
+# TODO: make the following work; for now they fail because the conformance
+# tests want to use isapprox on padic elements, but no such method exists
```

Comment:
Sorry about this. The conformance tests did not pass with BigFloat and ==. I think it is better to remove BigFloat as a conforming Ring because the == there does not conform to our standards.

Reply:
No worries. You already explained what went on. I just left this in as a reminder to take care of it in the future.